crates/sui-framework/packages/deepbook/sources/clob_v2.move

```diff
@@ -484,6 +490,19 @@ module deepbook::clob_v2 {
                     skip_order = true;
                     custodian::unlock_balance(&mut pool.base_custodian, maker_order.owner, maker_order.quantity);
                     emit_order_canceled<BaseAsset, QuoteAsset>(pool_id, maker_order);
+                    let canceled_order_event = OrderCanceled<BaseAsset, QuoteAsset> {
```

Comment:
Not a big deal, but it might be less error-prone to change `emit_order_canceled` to return a copy of the event instead of recreating it here